.changes/unreleased/tinymce-TINY-10596-2024-02-07.yaml

```diff
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Added
+body: Added focus behavior for CustomEditor.
```

Comment:
If you move this into the spec you can change this to be `Added onFocus callback for CustomEditor dialog component.` or something like that.

Reply:
changed